### PR TITLE
Fix location issue

### PR DESCRIPTION
**Pull request by BreakSecurity**

### CODE REVIEW

File: requests/Coupon.php [hunk 1]
```diff
@@ -29,6 +29,7 @@ public function rules()
             ['order_restriction', 'igniter.coupons::default.label_order_restriction', 'integer'],
             ['status', 'admin::lang.label_status', 'boolean'],
             ['locations.*', 'admin::lang.column_location', 'integer'],
+            ['locations', 'admin::lang.column_location', 'required'],
```
Comment (sampoyigi) [inline requests/Coupon.php:31]: For the sake of consistency, the `locations` rule should come before `locations.*`

**Review by sampoyigi — CHANGES_REQUESTED** [review]

**Review by sampoyigi — APPROVED** [review]

**Comment by sampoyigi** [comment]
Thank you!

**Comment by sampoyigi** [comment]
Ok reverting this as we want users to be able to create non-locationable coupons.

**Comment by BreakSecurity** [comment]
I'm not sure user will see the coupons with no location attached, but I agree

**Comment by sampoyigi** [comment]
Yup, just fixed that here tastyigniter/TastyIgniter@13f7e82a9ed801d20765af84b2dd999f4291a926 and ffc18e116aae37c35517081867adb8684bdf4528

**Pull request merged**